mitodl/mit-open
# breadcrumbs component

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/4593

### Description (What does it do?)
This PR adds a new `Breadcrumbs` component to `ol-components`. This component takes a variant of "light" or "dark," list of dicts in the `ancestors` property, each with an `href` and `label` property as well as a value for `current` which is just a string labeling the current page. The component is implemented on the various pages that use breadcrumbs.

### Screenshots (if appropriate):
![image](https://github.com/mitodl/mit-open/assets/12089658/f5e3733c-617a-4520-8deb-43efdecbe7f1)
![image](https://github.com/mitodl/mit-open/assets/12089658/f21663c6-be7a-40f2-9291-a4fe4446c5a8)
![image](https://github.com/mitodl/mit-open/assets/12089658/91c8da4a-ee88-4fa2-8c34-ff44cf812da1)
![image](https://github.com/mitodl/mit-open/assets/12089658/52561718-96a2-4a99-b27b-4c0e283a9d41)

### How can this be tested?
 - Spin up `mit-open` on this branch
 - Visit http://localhost:8063/units/
 - Verify that the breadcrumbs load properly, and that the Home link goes back home
 - Go back to the units page
 - Click on any of the cards to visit a unit channel page
 - Verify that clicking the MIT Units breadcrumb brings you back to the units page
 - View some of the other pages that had their breadcrumbs updated:
   - About Us (http://localhost:8063/about/)
   - Privacy Policy (http://localhost:8063/privacy/)
   - Terms of Service (http://localhost:8063/terms/)
   - Departments (http://localhost:8063/departments/)
   - Topics (http://localhost:8063/topics/)
 - View the storybook for the new component: http://localhost:6006/?path=/story/ol-components-breadcrumbs--story
